README.md

```diff
@@ -1,3 +1,106 @@
+but **should not** be used for production.
+
+### Supported Platforms
+
+| Platform        | Included in Operator | Tested in CI |
+| --------------- | -------------------- | ------------ |
+| AWS             | Yes                  | Yes          |
+| Azure           | No                   |              |
+| GCP             | No                   |              |
+| OpenStack       | Yes                  | No           |
+| vSphere         | No                   |              |
+
+## Deploying and Running CCCMO
+
+The CCCMO deploys controllers which provide a central core component of
+Kubernetes. As such, its deployment and operation is highly sensitive to
+cluster bootstrapping and initial payload deployment. In general, it is best
+to allow the OpenShift installer to manage its operation.
+
+To better understand how this operator is deployed, please see the
+[`manifest`](/manifest) directory. It contains a series of Kubernetes
+YAML manifests which are deployed by the CVO during installation.
+Additionally,
+[this OpenShift enhancement(currently in review)](https://github.com/openshift/enhancements/pull/463/)
+provides detailed information about how the operator will be deployed,
+operated, and upgraded.
+
+## Development
+
+**Prerequisites**
+* Go language 1.16+
+* GNU Make
+
+All development related tasks can be run through the `Makefile`. Supplemental
+scripts can be found in the `hack` directory.
+
+If you do not have the necessary tools for building, but do have access to
+Podman or Docker, you may use the `hack/container-run.sh` script to run
+Makefile targets in a container. See the `container-run.sh` file for usage
+instructions.
+
+### Building the Operator
+
+To build the operator binary, and run related linting tests, type `make` or
+`make build` from the root of the project.
+
+### Vendoring Dependencies
+
+After adding or updating dependencies in the `go.mod` file, run `make vendor`
+to ensure that all new dependencies are added to the `vendor` directory. It
+is also useful to run `make vet` to ensure that no build-time errors have
+been introduced during the vendor process.
+
+### Running Tests
+
+The CCCMO has multiple levels of testing: unit tests, and end to end (e2e)
+functionality tests. As a developer you should run the unit tests locally
+to ensure that your changes do not break the tests. Although running the
+e2e tests manually can be a rewarding experience, it is also complicated to
+configure and maintain. For these reasons it is often better to let the
+continuous integration systems run the e2e tests automatically for your
+on pull requests.
+
+#### Local Unit Tests
+
+To invoke the unit tests, run `make unit`. If you wish to also run the
+code generation and verification steps, run `make test`.
+
+*Note When running the local unit tests you may see errors about a missing
+`etcd` binary. These tests expect a slightly different configuration and
+are known to fail on local hosts.*
```

Comment:
Are these kubebuilder tests? Can we link to any instructions on how to set up the etcd binary?

Reply:
yeah, i would love to figure this out. any link?

Reply:
Maybe something like https://book.kubebuilder.io/reference/envtest.html?

Reply:
thanks! i'll investigate a little more